php/docker/templates/nginx-conf.d/10-lua-init.conf

```
@@ -12,3 +13,20 @@ init_by_lua_block {
     google_credentials = gcs.init()
 }
 
+init_worker_by_lua_block {
+    prometheus = require("prometheus").init("prometheus_metrics")
+    metric_requests = prometheus:counter(
+        "nginx_http_requests_total", "Number of HTTP requests", {"host", "status", "cache_status"}
+    )
+    metric_request_time = prometheus:histogram(
+        "nginx_http_request_duration_seconds", "HTTP request request duration", {"host"}
+    )
+    metric_connections = prometheus:gauge(
+        "nginx_http_connections", "Number of HTTP connections", {"state"}
+    )
+}
+
+log_by_lua_block {
+    metric_requests:inc(1, {ngx.var.server_name, ngx.var.status, string.lower(ngx.resp.get_headers()["x-cache-fetch"] or "disabled")})
```

Let's insert the site name and split the line into multiple lines. This should be done for all metrics.

```suggestion
    metric_requests:inc(1, {
        "{{ default "-" .Env.STACK_SITE_NAME }}",
        ngx.var.server_name, ngx.var.status,
        string.lower(ngx.resp.get_headers()["x-cache-fetch"] or "disabled")
    })
```